portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +9,101 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div>
+        <h1>Sofia Vega</h1>
+        <div>
+          <!--Shows a picture of me-->
+          <div id="crop-circle">
+            <img src="images/foto.jpg" id="my-photo">
+          </div>
+          <div>
+            <!--This div show icons for my github, linkedin and cv links-->
+            <div id="links">
+              <a href="https://github.com/SofiaVega">
+                <img src="images/github.png" height="32" width="32">
+              </a>
+              <a href="https://www.linkedin.com/in/sofia-vega-765645150/">
+                <img src="images/ln.png" height="32" width="32">
+              </a>
+              <a href="https://drive.google.com/file/d/1IZ7uzrvR5h-p26RDKwslGRcizleu-QLs/view?usp=sharing">
+                <img src="images/cv.png" height="32" width="32">
+              </a>
+            </div>
+          </div>
+        </div>
+        <br>
+        <!--This is a line with some basic info-->
+        <div id="short-info">
+          <span>STEP Intern</span>
+          <span>CS Student</span>
+          <span>ITESM</span>
+          <span>Culiacan/Monterrey</span>
+        </div>
+        <p>Hello! My name is Sofia and I’m a Computer Science Student at Tec de Monterrey. I’m a part of my major’s student board and enjoy helping other students develop their skills. In my free time I like writing short stories and painting.</p>
```

Comment:
nit: instead of making this one long line, maybe split it up for legibility? e.g.
```
<p>
    Hello! My name is Sofia...
    ... and enjoy helping other students...painting.
</p>
```

And same for others below.